cts-mongosh/query/array/all/simple.request.js

```diff
@@ -0,0 +1,14 @@
+db.runCommand({
```

Comment:
The formatting seems not indented. I guess when this is used in FerretDB, it will be formatted by our tooling so it's okay I guess. :thinking: